### PR TITLE
mapmerger readme

### PR DESCRIPTION
**Pull request by optimumtact**
basic guide and some troubleshooting

### CODE REVIEW

File: tools/mapmerge/readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+However - to do all this is going to require you to put some elbow grease into understanding the map merger tool.
+
+If you have difficulty using these tools, ask for help in #coderbus
+
+#Using the tools#
+
+##1. Install Python 3.5 or greater##
+If you don't have Python already installed it can be downloaded from: https://www.python.org/downloads/ - make sure you grab the latest python 3, again, it must be 3.5 or greater
+##2. PATH Python##
+This step is mostly applicable to windows users, you must make sure you ask the windows installer to add python to your path, [like shown in this example screenshot](https://file.house/DA6H.png)
```
Comment (Jordie0608) [inline tools/mapmerge/readme.md]: Probably a good idea to include how to manually add it to path ref. http://www.computerhope.com/issues/ch000549.htm or http://superuser.com/questions/143119/how-to-add-python-to-the-windows-path perhaps.

File: tools/mapmerge/readme.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+##3. Prepare Maps##
+Run "Prepare Maps.bat" in the tools/mapmerge/ directory.
+##4. Edit your map##
+make your changes to the map here. Remember to save them!
```
Comment (Jordie0608) [inline tools/mapmerge/readme.md]: Critical capitalization error detected.

File: tools/mapmerge/readme.md (hunk 1 above)
Comment (optimumtact) [inline tools/mapmerge/readme.md]: IF you got the point of needing to manually add it you messed up at following basic instructions anyway so there's no point even trying to help you

Comment (Jordie0608) [inline tools/mapmerge/readme.md]: I more meant for if someone already has Python installed without it being in the path and wants to add it while reading these instructions.

Comment (xxalpha) [inline tools/mapmerge/readme.md]: If you fail to do it during installation just uninstall and install again because 
>manually editing windows path

Comment (Jordie0608) [inline tools/mapmerge/readme.md]: I found it easy to do =\

Comment (xxalpha) [inline tools/mapmerge/readme.md]: I've done it before, but for this I wasn't able to no matter what, for reasons unknown.

It's far easier to just reinstall it properly, for experienced and less experienced users.

**Comment by Incoming5643** [comment]
Fixes #18978

**Comment by AnturK** [comment]
Replace or delete the old one please.

**Comment by optimumtact** [comment]
Should be good now

**Pull request merged**